torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -537,15 +537,13 @@ def prepare_for_partitioner(mod, num_primals, num_fw_outputs):
             and node.target is invoke_subgraph
             and isinstance(node.args[1], str)
         ):
-            identifier = (
-                node.args[1].replace("___forward", "").replace("___backward", "")
-            )
+            identifier = node.args[1].replace("_forward", "").replace("_backward", "")
 
             # NB: This is done in a separate if else condition because we early
             # return if the partitioning_done is True.
-            if node.args[1].startswith("___forward"):
+            if node.args[1].startswith("_forward"):
```

Comment:
"___" is reserved for [Name Mangling](https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name) in python. It leads to `name '_Runner___forward_invoke_subgraph_0_post_graph' is not defined` [error](name '_Runner___forward_invoke_subgraph_0_post_graph' is not defined). So it might be better to use single underscore.

Reply:
Fixed in #152581